var/spack/repos/builtin/packages/perl-parsetemplate/package.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
```

Comment:
FYI. Don't have to change this to 2024 now since the CI checks allow packages to be submitted with the prior year but it will be an issue in future years (assuming the check remains).